backend/src/main/java/org/cryptomator/hub/license/LicenseHolder.java

```diff
@@ -81,7 +93,19 @@ public long getAvailableSeats() {
 		return Optional.ofNullable(license) //
 				.map(l -> l.getClaim("seats")) //
 				.map(Claim::asLong) //
-				.orElse(CommunityLicenseConstants.SEATS);
+				.orElseGet(this::getNoLicenseSeats);
+	}
+
+	public long getNoLicenseSeats() {
+		if (!managedInstance) {
+			return CommunityLicenseConstants.SEATS;
+		} else {
+			return ManagedInstanceNoLicenseConstants.SEATS;
+		}
+	}
```

Comment:
How about renaming `CommunityLicenseConstants` to `SelfHostedNoLicenseConstants`? And also remove `Instance` in `ManagedInstanceNoLicenseConstants`?